src/codemagic/apple/app_store_connect/versioning/review_submissions.py

```diff
@@ -99,3 +100,11 @@ def modify(
             json=payload,
         ).json()
         return ReviewSubmission(response["data"])
+
+    def list_items(self, review_submission: Union[LinkedResourceData, ResourceId]):
```

Comment:
Return type?

Reply:
Yes! Somehow I forgot about it.

Reply:
Added in [90dd967](https://github.com/codemagic-ci-cd/cli-tools/pull/380/commits/90dd967934421ab7cc02fb6d7853127ac02f70ca).